<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Client</class>
 <widget class="QDialog" name="Client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1241</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 62);
background-image: url();</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_serch">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>60</y>
     <width>461</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
background-image: url();</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_serch">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>58</y>
     <width>55</width>
     <height>48</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/Search.png);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_moneybags">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>40</y>
     <width>81</width>
     <height>68</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/wallet.png);
background-color: rgb(0,0,61);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_logout">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>50</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0,0,61);
background-image: url(:/new/prefix1/log out.png);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Cart">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>40</y>
     <width>93</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 61);
background-image: url(:/new/prefix1/cart.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_commodity">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>120</y>
     <width>721</width>
     <height>551</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
background-image: url();</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="autoScrollMargin">
    <number>4</number>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::DoubleClicked</set>
   </property>
   <property name="iconSize">
    <size>
     <width>300</width>
     <height>198</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>351</width>
     <height>551</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
background-image: url();</string>
   </property>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>351</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>-3</x>
      <y>260</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();</string>
    </property>
    <property name="text">
     <string>MAX :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();</string>
    </property>
    <property name="text">
     <string>MIN :</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>330</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>400</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_coler">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>218</y>
      <width>201</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(209, 209, 209);
background-image: url();</string>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Red</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Blue</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Yellow</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Other</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_brand">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>290</y>
      <width>201</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(209, 209, 209);
background-image: url();</string>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Nike</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Addias</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Puma</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Other</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_Weight">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>360</y>
      <width>201</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(209, 209, 209);
background-image: url();</string>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>0-1000     gr</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1000-5000  gr</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5000&gt;     gr</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_type">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>430</y>
      <width>201</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(209, 209, 209);
background-image: url();</string>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Equipment</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Clothes and shoes</string>
     </property>
    </item>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>470</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_sex">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>500</y>
      <width>201</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(209, 209, 209);</string>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Men</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Women</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Other</string>
     </property>
    </item>
   </widget>
   <widget class="QSpinBox" name="spinBox_max">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();</string>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>100000000</number>
    </property>
    <property name="value">
     <number>100000000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_min">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();</string>
    </property>
    <property name="maximum">
     <number>90</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();</string>
    </property>
    <property name="text">
     <string>Price:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>213</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();</string>
    </property>
    <property name="text">
     <string>Color:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>293</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();</string>
    </property>
    <property name="text">
     <string>brand:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>355</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();</string>
    </property>
    <property name="text">
     <string>Weight:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();</string>
    </property>
    <property name="text">
     <string>Product Type:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();</string>
    </property>
    <property name="text">
     <string>Gender:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();</string>
    </property>
    <property name="text">
     <string>Filters</string>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>351</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>181</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/logo.png);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_price">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>80</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url();
background-color: rgb();
color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>65225</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>200</y>
     <width>111</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/BitcoinAd.gif);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>70</y>
     <width>52</width>
     <height>42</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/refresh.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="shortcut">
    <string>Ctrl+F5</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
